Mail/Views/New Message/ComposeMessageView.swift

```diff
@@ -242,12 +254,18 @@ struct ComposeMessageView: View {
 
     // MARK: - Func
 
+    /// Something to dismiss the view regardless of presentation context
+    private func dismissMessageView() {
+        dismissModal()
```

Comment:
We do this so that it works in the share extension ?

Reply:
Yes, you pointed me toward using a `@Environment(\.dismissModal)` to catch dismiss events in extension and in app. Maybe I can tweak it some more to your liking ?